YifanLiao.md

```diff
@@ -1,37 +1,43 @@
-  - by design on woodduck
-    - [INVALID] [Bug 1095400 - [Woodduck][v2.0][Settings]The unlock sim pin button in "SIM Security" view is unavailable.]
-  - can't reproduce
-    - [WORKSFORME] [Bug 1106853 - [Window Management] Unable to set device to Landscape mode in Browser and Gallery.]
-  - duplicate of 1061858, uplift request submitted
-    - [DUPLICATE] [Bug 1095353 - [Wifi]The checked Wifi usage will be unchecked automatically after you exit and launch data usage again.]
   - [WIP] [Bug 1090680 - [Window Management]The card view for browser shows a black page.]
-  - APZ related
-    - [WIP] [Bug 1090690 - [Woodduck][v2.0][Email] You have to tap the delete button twice if you want to delete the AOL/Hotmail/163 Mail/Gmail account. ]
-  - can't reproduce
-    - [WIP] [Bug 1113061 - [Email]You can't select a picture attachment for email after unplug usb.]
+  - r?
+    - [WIP] [Bug 1108984 - [Woodduck][Dialer]The "No missed message" info will be displayed in "All" lab after deleting the missed call log in "All".]
+  - telephony issue
+    - [DUPLICATE] [Bug 1116642 - [FFOS2.0][Woodduck][Call]There is no missed call information when receive a missed call during active call.]
+  - r+, approval-b2g-v2.0+
+    - [WIP] [Bug 1117463 - [Browser]User can't slide down to view tabs after adding some tabs in Browser.]
+  - no 'open-app' event, need platform support
+    - [WIP] [Bug 1115640 - [Midori 2.0][FFOS2.0][Woodduck][Usage] Can't open Usage when tap nortification of data limit.]
+  - audio competition, platform issue
+    - [WIP] [Bug 1118148 - [FFOS2.0][Woodduck][Download]The audio play in browser cannot pause when play the downloads list audio.]
+  - r?
+    - [WIP] [Bug 1100754 - [FFOS2.0][Woodduck][Browser]Top sites display incompletely on landscape mode]
+  - duplicte of 946380, passed it to tech evangelism team
+    - [DUPLICATE] [Bug 1119139 - [FFOS2.0][Woodduck][wap]browser can't open orange websites.]
 * [System browser]
   - [WIP] [Bug 1088178 - When in fullscreen mode, the context menu is overlayed by the fullscreen video]
+* [Study]
+  - system browser
+    - started
 
 This Week
 
-* [TV]
-  - follow up
-    - [WIP] [Bug 1100780 - [Stingray] A button of select element does not work]
-    - [WIP] [Bug 1100807 - [Stingray] Capturing wrong screenshot by getScreenshot() at some pages]
 * [Partner support]
-  - duplicate of 1073340, made v2.0 patch, r?
+  - duplicate of 1073340, made v2.0 patch, r+, approval-b2g-v2.0-
     - [DUPLICATE] [Bug 1095364 - [Flame][Woodduck][v2.0&2.1][Settings]Can't set "0.14GB" as data used limit in Usage settings.]
-  - [WIP] [Bug 1090680 - [Window Management]The card view for browser shows a black page.]
+  - r?
+    - [WIP] [Bug 1108984 - [Woodduck][Dialer]The "No missed message" info will be displayed in "All" lab after deleting the missed call log in "All".]
+  - no 'open-app' event, need platform support
```

Comment:
We'd like to know "why" there is no open-app event, if you can spend more time looking into it, that will be great.